### PR TITLE
Close inspector on deployer bar commit

### PR DESCRIPTION
**Pull request by hatched**
When deploying commits the inspector needs to be closed and the added-services component shown.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA
- Deploy a service: 
```
app.get('charmstore').getEntity('precise/python-django', function(entity) { app.deployService(entity[0]); });
```
- Open it's inspector
- Open commit and deploy the deployer bar.
- The added services list should be shown

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2998/
Test PASSed.

File: jujugui/static/gui/src/test/test_environment_view.js [hunk 1]
```diff
@@ -1538,7 +1538,7 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
       view.destroy();
     });
 
-    describe.only('onboarding integration with the environment', function() {
+    describe('onboarding integration with the environment', function() {
```
Comment (makyo) [inline jujugui/static/gui/src/test/test_environment_view.js]: Whoops, thanks!

**Comment by makyo** [comment]
Code :+1: QAing now

**Comment by makyo** [comment]
QA okay

**Comment by anthonydillon** [comment]
Code looks good :+1:

**Comment by hatched** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1044

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**